### PR TITLE
Show HCA application status on static pages

### PR DESCRIPTION
**Pull request by cehsu**
### Health Care Benefits

localhost-3001-health-care- 1

### Health Care Benefits Eligibility

localhost-3001-health-care-eligibility-

### Health Care Application Process

localhost-3001-health-care-apply-

### CODE REVIEW

File: content/components/navigation-links-list.html [hunk 1]
```diff
@@ -57,6 +57,9 @@ <h4 class="va-nav-linkslist-title">{{ link.title }}</h4>
               </a>
             </li>
           {% endif %}
+          {% if link.sibling !== false %}
```
Comment (jbalboni) [inline content/components/navigation-links-list.html]: What's this?

File: content/pages/health-care/index.md [hunk 2]
```diff
@@ -16,6 +21,9 @@ majorlinks:
     - url: /health-care/apply/
       title: Application Process
       description: Apply online now, or find out how to apply in person, by phone, or by mail.
+      sibling: true
```
Comment (jbalboni) [inline content/pages/health-care/index.md]: I think on the pension page we put the status above the major links, rather than in them.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu Hi, Claire, a few tweaks from me:
The HC overview page:
-- can we move the message to above the majorlinks? The placement looks off embedded in the links. 
--the Continue Your Application button is missing

The HC Application process page:
--Delete the line "You can start your online application right now" when the application status message pops up.

File: content/pages/health-care/index.md (hunk 2 above)
Comment (cehsu) [reply content/pages/health-care/index.md]: Thanks @jbalboni @peggygannon for pointing this out. I had made this change based on @stacyhlinn [comment](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5268#issuecomment-336021727) on the original issue. Thanks all for confirming which way the card should be displayed.

File: content/components/navigation-links-list.html (hunk 1 above)
Comment (cehsu) [reply content/components/navigation-links-list.html]: This should be `{% if link.sibling %}`, and has been changed.

**Comment by stacyhlinn** [comment]
@peggygannon The reason we decided to move it below the application process is because there is a homeless information card there already. I was concerned about all these message boxes stacking on top of each other. If you think that is not a valid concern, I'm find moving it up, and be consistent with what we have already done in pension.

**Comment by cehsu** [comment]
@stacyhlinn @peggygannon @jbalboni thanks for your comments. I have provided a screenshot below of what the Health Care Benefits screen would look like with the SIP card located in the same place as on the Pension Benefits page. 
localhost-3001-health-care-

**Comment by cehsu** [comment]
Thanks @peggygannon, kindly find a screenshot of the updated Health Care Application Process page (on which the "You can start your online application right now" text is removed as the SIP card is being added):
localhost-3001-health-care-apply-

**Review by jbalboni — APPROVED** [review]
LGTM from a code perspective. It's not clear if there's still more copy/design work to do.

**Review by stacyhlinn — COMMENTED** [review]
I'm good with the SiP card sitting above all the links.

File: content/pages/health-care/apply.md [hunk 3]
```diff
@@ -64,11 +69,11 @@ Or, [find your state’s Veterans agency](https://www.va.gov/statedva.htm).
 <h3 itemprop="name">Ready to apply?</h3>
 <div itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
 <div itemprop="text">
-
-You can start your online application right now.
-
-<a class="usa-button-primary va-button-primary" href="/health-care/apply/application/">Apply for Health Care Benefits</a>
-
+<div itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
```
[the remark this answers is not in the document]
Comment (cehsu) [reply content/pages/health-care/apply.md]: Thanks Jeff, nice catch. This has been changed.

File: content/pages/health-care/eligibility.md [hunk 4]
```diff
@@ -62,9 +67,11 @@ If none of the above apply to you, you may still qualify for care based on your
 
 ### Ready to apply?
 
-<a class="usa-button-primary va-button-primary" href="/health-care/apply/application/">Apply for Health Care Benefits</a>
+<div id="react-applicationStatus">
+  <a class="usa-button-primary va-button-primary" href="/health-care/apply/application/">Apply for Health Care Benefits</a>
+</div>
 
-[Learn about the application process](/health-care/apply/).
+[Learn about the application process](/pension/apply/).
```
[the remark this answers is not in the document]
Comment (cehsu) [reply content/pages/health-care/eligibility.md]: Thanks Jeff, nice catch, this has been changed.

File: src/js/common/components/createApplicationStatus.jsx [hunk 5]
```diff
@@ -2,20 +2,20 @@ import React from 'react';
...
 import { Provider } from 'react-redux';
 
-export default function createPensionApplicationStatus(store) {
-  const root = document.getElementById('react-pensionApplicationStatus');
-
+export default function createApplicationStatus(store, form) {
+  const root = document.getElementById('react-applicationStatus');
   if (root) {
     import(
       /* webpackChunkName: "pension-application-status" */
```
[the remark this answers is not in the document]
Comment (cehsu) [reply src/js/common/components/createApplicationStatus.jsx]: Thanks Jeff, this has been changed to "application-status", does it need to be updated anywhere else?

**Review by peggygannon — APPROVED** [review]
LGTM

File: src/js/common/components/createApplicationStatus.jsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (jbalboni) [reply src/js/common/components/createApplicationStatus.jsx]: Nope, just there.

File: src/js/no-react-entry.js [hunk 6]
```diff
@@ -23,6 +24,18 @@ if (wizardPages.has(location.pathname)) {
 }
 
 const store = createCommonStore();
-
 createLoginWidget(store);
-createPensionApplicationStatus(store);
+if (pensionPages.has(location.pathname)) {
+  createApplicationStatus(store, {
+    formId: '21P-527EZ',
+    applyText: 'Apply for Veterans Pension'
```
Comment (cehsu) [inline src/js/no-react-entry.js]: @peggygannon should the Pension apply button also include the word "benefits".

Comment (peggygannon) [reply src/js/no-react-entry.js]: @cehsu Yes, thanks for checking. Yes, I would recommend: Apply for Veterans Pension Benefits.

Comment (cehsu) [reply src/js/no-react-entry.js]: Thanks @peggygannon, I'll loop that into the next set of similar changes.

**Pull request merged**